modules/apps/commerce/commerce-product-definitions-web/src/main/java/com/liferay/commerce/product/definitions/web/internal/editor/configuration/CommerceCKEditorConfigContributor.java

```diff
@@ -0,0 +1,203 @@
+			).setMVCPath(
+				"/view.jsp"
+			).setPortletMode(
+				PortletMode.VIEW
+			).setWindowState(
+				LiferayWindowState.POP_UP
+			).buildString()
+		).put(
+			"aiCreatorPortletNamespace",
+			() -> _portal.getPortletNamespace(
+				"com_liferay_ai_creator_openai_web_internal_portlet_" +
+					"AICreatorOpenAIPortlet")
+		).put(
+			"extraPlugins", extraPlugins.concat(",aicreator")
+		).put(
+			"isAICreatorOpenAIAPIKey",
+			() -> {
+				try {
+					if (Validator.isNotNull(
+							_aiCreatorOpenAIConfigurationManager.
+								getAICreatorOpenAIGroupAPIKey(
+									themeDisplay.getCompanyId(),
+									themeDisplay.getScopeGroupId()))) {
+
+						return true;
+					}
+				}
+				catch (ConfigurationException configurationException) {
+					if (_log.isDebugEnabled()) {
+						_log.debug(configurationException);
+					}
+				}
+
+				return false;
+			}
+		).put(
+			"removePlugins", "elementspath"
+		).put(
+			"toolbar_editInPlace", toolbarSimpleJSONArray
+		).put(
+			"toolbar_email", toolbarSimpleJSONArray
+		).put(
+			"toolbar_liferay", toolbarSimpleJSONArray
+		).put(
+			"toolbar_liferayArticle", toolbarSimpleJSONArray
+		).put(
+			"toolbar_phone", toolbarSimpleJSONArray
+		).put(
+			"toolbar_simple", toolbarSimpleJSONArray
+		).put(
+			"toolbar_tablet", toolbarSimpleJSONArray
+		).put(
+			"toolbar_text_advanced", _getToolbarTextAdvancedJSONArray()
+		).put(
+			"toolbar_text_simple", _getToolbarTextSimpleJSONArray()
+		);
+	}
+
+	private JSONArray _getToolbarSimpleJSONArray() {
+		return JSONUtil.putAll(
+			toJSONArray("['Undo', 'Redo']"),
+			toJSONArray("['Styles', 'Bold', 'Italic', 'Underline']"),
+			toJSONArray("['NumberedList', 'BulletedList']"),
+			toJSONArray("['Link', Unlink]"),
+			toJSONArray("['Table', 'ImageSelector', 'VideoSelector']")
+		).put(
+			() -> {
+				if (AudioProcessorUtil.isEnabled()) {
+					return toJSONArray("['AudioSelector']");
+				}
+
+				return null;
+			}
+		).put(
+			toJSONArray("['AICreator']")
+		);
+	}
+
+	private JSONArray _getToolbarTextAdvancedJSONArray() {
+		return JSONUtil.putAll(
+			toJSONArray("['Undo', 'Redo']"), toJSONArray("['Styles']"),
+			toJSONArray("['FontColor', 'BGColor']"),
+			toJSONArray("['Bold', 'Italic', 'Underline', 'Strikethrough']"),
+			toJSONArray("['RemoveFormat']"),
+			toJSONArray("['NumberedList', 'BulletedList']"),
+			toJSONArray("['IncreaseIndent', 'DecreaseIndent']"),
+			toJSONArray("['IncreaseIndent', 'DecreaseIndent']"),
+			toJSONArray("['Link', Unlink]")
+		).put(
+			toJSONArray("['AICreator']")
+		);
+	}
+
+	private JSONArray _getToolbarTextSimpleJSONArray() {
+		return JSONUtil.putAll(
+			toJSONArray("['Undo', 'Redo']"),
+			toJSONArray("['Styles', 'Bold', 'Italic', 'Underline']"),
+			toJSONArray("['NumberedList', 'BulletedList']"),
+			toJSONArray("['Link', Unlink]")
+		).put(
+			toJSONArray("['AICreator']")
+		);
+	}
+
+	private boolean _isAICreatorChatGPTGroupEnabled(
+		long companyId, long groupId) {
+
+		try {
```

Comment:
Is a FF needed or is it already inside the isAICreatorChatGPTGroupEnabled method?

Reply:
added it